src/enum_net.cpp

```diff
@@ -377,6 +377,38 @@ namespace libtorrent
 		return false;
 	}
 
+#ifdef TORRENT_WINDOWS
+	HMODULE GetIPHelperHandle()
+	{
+		static bool handle_checked = false;
+		static HMODULE handle = 0;
+		if (!handle_checked)
```

Comment:
I guess the point is to cache the failure, is it? if we fail to load the library, we don't want to retry every time it's called